recipes/xarray-behave/bld.bat

```diff
@@ -0,0 +1,8 @@
+# We need to turn pip index back on because Anaconda turns
+# it off for some reason.
+set PIP_NO_INDEX=False
+set PIP_NO_DEPENDENCIES=False
+set PIP_IGNORE_INSTALLED=False
+
+"%PYTHON%" -m pip install xarray-behave[gui] -vv --no-dependencies
```

Comment:
This is not allowed in conda-forge. Those flags are set for a reason. Also, the optional dependencies for the gui must either be in the basic package, or you need to create a multiple outputs package with `xarray-behave` and `xarray-behave-gui` outputs.